config/default_replication_exclude_tables.yml

```diff
@@ -43,7 +43,6 @@
 - metrics_21
 - metrics_22
 - metrics_23
-- metric_rollups
```

Comment:
Is this technically a bug fix?

Reply:
Yeah, do you want me to break this out into a separate PR?

Reply:
Extracted this out into https://github.com/ManageIQ/manageiq/pull/18702